### ISSUE TITLE
SVG Brands not loading while Solids Are

### ISSUE THREAD

**Issue by nnagewad** [FortAwesome/vue-fontawesome#82]
Hello,

I am trying to place some branded SVGs into a project; however, it is not working... I was successful with Solids though.

I have a working example up on codesandbox as suggested.
https://codesandbox.io/s/nw44jz37q4

**Comment by ebisbe** [comment]
@robmadole why did you close this issue? I have the same problem. Solid icons are working but brands aren't... Any clue?

```
import Vue from 'vue'
// the component
import { library } from '@fortawesome/fontawesome-svg-core'
import { faCoffee, faCog } from '@fortawesome/free-solid-svg-icons'
//import brands from '@fortawesome/fontawesome-free-brands'
import { fa500px } from '@fortawesome/free-brands-svg-icons'
import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'

library.add(faCoffee, fa500px, faCog)

Vue.component('font-awesome-icon', FontAwesomeIcon)
```

**Comment by ebisbe** [comment]
Oks, I have found that it's closed because it has been explained in  #81
